modules/actions/discard_tags.js

```diff
@@ -13,7 +13,8 @@ export function actionDiscardTags(difference, discardTags) {
 
       for (let i = 0; i < keys.length; i++) {
         const k = keys[i];
-        if (discardTags[k] || !entity.tags[k]) {
+        const v = entity.tags[k];
+        if (discardTags[k] === true || (typeof discardTags[k] === 'object' && discardTags[k][v]) || !entity.tags[k]) {
```

Comment:
Btw: At first, I was wondering what the `|| !entity.tags[k]` is doing here… Then I realized that this is responsible for actually deleting a tag when one fully clears the value of a tag, e.g. in the raw tag editor (i.e. `entity.tags[k] === ''`). :sweat_smile: